elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -609,6 +610,22 @@ elytron.simple-role-decoder.remove=Remove operation for the RoleDecoder
 # Attributes
 elytron.simple-role-decoder.attribute=The name of the attribute from the identity to map directly to roles.
 
+elytron.source-address-role-decoder=Definition of a RoleDecoder that maps roles based on the IP address of a remote client.
```

Comment:
should the descriptions distinguish between the generic `RoleDecoder` term vs specific `source-address-role-decoder` & `aggregate-role-decoder`?

Reply:
These descriptions mention the generic `RoleDecoder` to be consistent with the other `*-role-decoder` resource descriptions (e.g., `simple-role-decoder`, `custom-role-decoder`). The `*-role-decoder` resources all provide the `RoleDecoder` capability. So we tend to use the generic term to indicate that these resources can be referenced anywhere in the Elytron subsystem where a `role-decoder` can be specified.